### PR TITLE
Flip cvr upload

### PR DESCRIPTION
**Pull request by scottferg**
Also a sweet hack to fix the container

### CODE REVIEW

**Comment by scottferg** [comment]
Well that didn't work so well

File: Dockerfile [hunk 1]
```diff
@@ -12,6 +12,7 @@ RUN apt-get install -y ca-certificates libvips37 libxml2-dev \
     && make install \
     && ldconfig
 RUN mkdir /etc/vip
+RUN ln -s /usr/lib/libvips.so.42 /usr/lib/libvips.so.38
```
Comment (foresmac) [inline Dockerfile]: Why is this necessary?

File: .drone.yml [hunk 2]
```diff
@@ -25,5 +26,4 @@ script:
     - cd fetch && go test -v -coverprofile=coverage.txt -covermode=count
     - cd ..
     - cat fetch/coverage.txt >> coverage.txt
-    - export CVR_TOKEN=$$cvrToken
-    - curl -F coverage=@coverage.txt "https://cvr.vokal.io/coverage?token=$CVR_TOKEN&commit=$GITHASH&coveragetype=gocover&removepath=vip/"
+    - curl -sSL https://cvr.vokal.io/upload | bash
```
Comment (foresmac) [inline .drone.yml]: This is one of those projects  that relies on remove path.

Comment (scottferg) [inline .drone.yml]: Yeah which is now configured within cvr

File: Dockerfile (hunk 1 above)
Comment (scottferg) [inline Dockerfile]: It's a hack. I believe dependencies in libvips are the culprit but this works fine as is currently.

Comment (foresmac) [inline Dockerfile]: Ok, I have it on the roadmap to update to a much newer version of `libvips` as well as using that different Go binding for it since it'll give us transparency support in PNGs and the ability to resize GIFs, among other things.

**Comment by jrit** [comment]
So, why did the coverage percent drop 37% here?

**Comment by scottferg** [comment]
I think I messed up how the report is generated. Whoops. It's not a cvr
issue.

**Comment by foresmac** [comment]
@jrit The other weird thing about the drop is the project _was_ at 32%, so if the change is in absolute it should be approx –23%, right?

**Comment by scottferg** [comment]
It's only uploading a fraction of the coverage files I wouldn't think too heavily on it. I'll fix it.

**Comment by scottferg** [comment]
Yup the coverage.txt output is mangled, not related to this PR or anything in cvr

**Comment by foresmac** [comment]
What's mangled about it? It looks correct to me.

**Comment by scottferg** [comment]
Look in the build logs

**Comment by foresmac** [comment]
Yeah, I'm looking it and it looks fine. The problem is here:
`Uploading ./fetch/coverage.txt to cvr...`

It's only uploading the `coverage.txt` found in the `fetch` package, not the concat'd version in the project root.

**Comment by scottferg** [comment]
Scroll up where I catted the coverage file. It's not formatted correctly. Works fine locally.

**Comment by foresmac** [comment]
Try changing line 28 in `drone.yml` to:
```sh
cat fetch/coverage.txt >> coverage.txt && rm fetch/coverage.txt
```

**Comment by scottferg** [comment]
That will help too but I'm more concerned at the formatting

**Comment by foresmac** [comment]
If you're talking about this, then it is correct:
```
$ cat fetch/coverage.txt >> coverage.txt
$ cat coverage.txt
mode: count
vip/main.go:44.19,49.12 3 0
vip/main.go:49.12,53.39 3 0
vip/main.go:57.3,57.77 1 0
vip/main.go:53.39,55.4 1 0
vip/main.go:57.77,59.4 1 0
vip/main.go:60.4,61.56 1 0
vip/main.go:61.56,63.4 1 0
vip/main.go:67.29,70.8 3 2
vip/main.go:70.8,72.3 1 1
vip/main.go:72.4,77.3 2 1
vip/main.go:80.13,86.16 6 1
vip/main.go:91.2,92.16 2 1
vip/main.go:97.2,101.21 4 1
vip/main.go:105.2,106.25 2 1
vip/main.go:113.2,114.24 2 1
vip/main.go:122.2,132.21 9 1
vip/main.go:86.16,89.3 2 1
vip/main.go:92.16,95.3 2 1
vip/main.go:101.21,103.3 1 1
vip/main.go:106.25,108.3 1 1
vip/main.go:108.4,111.3 2 0
vip/main.go:114.24,116.3 1 1
vip/main.go:116.4,118.17 2 0
vip/main.go:118.17,120.4 1 0
vip/main.go:135.13,139.16 3 0
vip/main.go:143.2,148.60 4 0
vip/main.go:152.2,153.70 1 0
vip/main.go:164.2,164.15 1 0
vip/main.go:174.2,182.20 6 0
vip/main.go:139.16,141.3 1 0
vip/main.go:148.60,150.3 1 0
vip/main.go:153.70,157.18 3 0
vip/main.go:161.4,161.27 1 0
vip/main.go:157.18,159.5 1 0
vip/main.go:164.15,166.17 2 0
vip/main.go:166.17,169.4 2 0
vip/main.go:169.5,171.4 1 0
vip/handler.go:44.31,47.2 2 0
vip/handler.go:49.71,54.16 4 7
vip/handler.go:59.2,61.34 2 7
vip/handler.go:73.2,74.23 2 7
vip/handler.go:78.2,78.21 1 7
vip/handler.go:83.2,83.35 1 6
vip/handler.go:87.2,87.9 1 5
vip/handler.go:54.16,57.3 2 0
vip/handler.go:61.34,63.12 2 10
vip/handler.go:63.12,69.9 5 3
vip/handler.go:74.23,76.3 1 3
vip/handler.go:78.21,81.3 2 1
vip/handler.go:83.35,85.3 1 1
vip/handler.go:90.59,97.2 5 6
vip/handler.go:99.58,101.12 2 1
vip/handler.go:106.2,106.75 1 1
vip/handler.go:101.12,103.3 1 0
vip/handler.go:103.4,105.3 1 1
vip/handler.go:109.59,112.45 2 0
vip/handler.go:116.2,116.30 1 0
vip/handler.go:112.45,115.3 2 0
vip/handler.go:119.65,120.23 1 0
vip/handler.go:124.2,128.45 2 0
vip/handler.go:133.2,137.16 4 0
vip/handler.go:141.2,142.119 2 0
vip/handler.go:120.23,122.3 1 0
vip/handler.go:128.45,131.3 2 0
vip/handler.go:137.16,139.3 1 0
vip/handler.go:145.59,146.24 1 5
vip/handler.go:151.2,153.33 2 5
vip/handler.go:169.2,174.16 5 4
vip/handler.go:178.2,180.16 2 4
vip/handler.go:185.2,187.22 2 4
vip/handler.go:196.2,205.45 5 4
vip/handler.go:146.24,149.3 2 0
vip/handler.go:153.33,160.3 4 0
vip/handler.go:160.4,160.33 1 5
vip/handler.go:160.33,167.3 4 1
vip/handler.go:174.16,176.3 1 0
vip/handler.go:180.16,183.3 2 0
vip/handler.go:187.22,189.13 2 0
vip/handler.go:189.13,191.4 1 0
vip/handler.go:191.5,193.4 1 0
vip/handler.go:205.45,208.3 2 1
vip/handler.go:211.57,214.2 2 0
vip/handler.go:216.82,217.49 1 6
vip/handler.go:217.49,219.17 2 5
vip/handler.go:222.3,222.23 1 5
vip/handler.go:226.3,232.17 6 5
vip/handler.go:235.3,237.45 2 5
vip/handler.go:219.17,221.4 1 0
vip/handler.go:222.23,224.4 1 0
vip/handler.go:232.17,234.4 1 0
vip/handler.go:239.4,241.17 2 1
vip/handler.go:245.3,248.17 4 1
vip/handler.go:252.3,258.45 5 1
vip/handler.go:241.17,243.4 1 0
vip/handler.go:248.17,250.4 1 0
mode: count
vip/fetch/context.go:18.80,20.16 2 0
vip/fetch/context.go:25.2,25.15 1 0
vip/fetch/context.go:20.16,23.3 2 0
vip/fetch/context.go:28.80,30.16 2 0
vip/fetch/context.go:35.2,35.18 1 0
vip/fetch/context.go:30.16,33.3 2 0
vip/fetch/context.go:38.76,40.2 1 0
vip/fetch/context.go:42.42,43.9 1 0
vip/fetch/context.go:50.2,50.37 1 0
vip/fetch/context.go:44.2,45.54 1 0
vip/fetch/context.go:46.2,47.52 1 0
vip/fetch/fetch.go:20.24,22.16 2 4
vip/fetch/fetch.go:26.2,26.17 1 2
vip/fetch/fetch.go:22.16,24.3 1 2
vip/fetch/fetch.go:29.52,34.22 3 0
vip/fetch/fetch.go:38.2,43.3 1 0
vip/fetch/fetch.go:34.22,36.3 1 0
vip/fetch/fetch.go:46.45,51.2 3 0
vip/fetch/fetch.go:53.81,55.9 2 0
vip/fetch/fetch.go:59.2,62.15 3 0
vip/fetch/fetch.go:68.2,69.16 2 0
vip/fetch/fetch.go:74.2,75.16 2 0
vip/fetch/fetch.go:85.2,86.18 2 0
vip/fetch/fetch.go:97.2,98.16 2 0
vip/fetch/fetch.go:102.2,102.12 1 0
vip/fetch/fetch.go:109.2,110.20 2 0
vip/fetch/fetch.go:55.9,57.3 1 0
vip/fetch/fetch.go:62.15,63.20 1 0
vip/fetch/fetch.go:63.20,65.4 1 0
vip/fetch/fetch.go:69.16,72.3 1 0
vip/fetch/fetch.go:75.16,77.17 2 0
vip/fetch/fetch.go:77.17,79.4 1 0
vip/fetch/fetch.go:80.4,83.3 2 0
vip/fetch/fetch.go:86.18,87.68 1 0
vip/fetch/fetch.go:92.3,92.17 1 0
vip/fetch/fetch.go:87.68,89.4 1 0
vip/fetch/fetch.go:89.5,91.4 1 0
vip/fetch/fetch.go:92.17,94.4 1 0
vip/fetch/fetch.go:98.16,100.3 1 0
vip/fetch/fetch.go:102.12,104.17 2 0
vip/fetch/fetch.go:104.17,106.4 1 0
vip/fetch/manip.go:17.39,19.16 2 10
vip/fetch/manip.go:23.2,24.16 2 10
vip/fetch/manip.go:28.2,28.30 1 10
vip/fetch/manip.go:19.16,21.3 1 0
vip/fetch/manip.go:24.16,26.3 1 0
vip/fetch/manip.go:29.2,30.13 1 2
vip/fetch/manip.go:31.2,32.13 1 1
vip/fetch/manip.go:33.2,34.12 1 1
vip/fetch/manip.go:35.2,36.11 1 6
vip/fetch/manip.go:41.66,43.16 2 0
vip/fetch/manip.go:47.2,50.16 3 0
vip/fetch/manip.go:54.2,54.43 1 0
vip/fetch/manip.go:58.2,59.15 2 0
vip/fetch/manip.go:68.2,68.27 1 0
vip/fetch/manip.go:43.16,45.3 1 0
vip/fetch/manip.go:50.16,52.3 1 0
vip/fetch/manip.go:54.43,56.3 1 0
vip/fetch/manip.go:60.2,61.34 1 0
vip/fetch/manip.go:62.2,63.35 1 0
vip/fetch/manip.go:64.2,65.35 1 0
vip/fetch/manip.go:71.64,73.16 2 0
vip/fetch/manip.go:77.2,86.12 2 0
vip/fetch/manip.go:103.2,104.16 2 0
vip/fetch/manip.go:108.2,108.34 1 0
vip/fetch/manip.go:73.16,75.3 1 0
vip/fetch/manip.go:86.12,90.17 3 0
vip/fetch/manip.go:94.3,96.57 2 0
vip/fetch/manip.go:100.3,100.33 1 0
vip/fetch/manip.go:90.17,92.4 1 0
vip/fetch/manip.go:96.57,98.4 1 0
vip/fetch/manip.go:104.16,106.3 1 0
vip/fetch/manip.go:111.67,113.16 2 0
vip/fetch/manip.go:116.2,116.21 1 0
vip/fetch/manip.go:120.2,121.36 2 0
vip/fetch/manip.go:125.2,125.26 1 0
vip/fetch/manip.go:113.16,115.3 1 0
vip/fetch/manip.go:116.21,118.3 1 0
vip/fetch/manip.go:121.36,123.3 1 0
```

**Comment by scottferg** [comment]
Okay lame it must be how drone is rendering it. It's all messed up on my end. Added your change, that should fix it

**Comment by scottferg** [comment]
Alrighty coverage is fixed. Weird that running it locally caught the correct `coverage.txt` before the other one.

**Comment by foresmac** [comment]
LGTM!
